### DISCUSSION TITLE
Bulk update fields

### DISCUSSION THREAD

**Discussion by michalkasp** [AntlerVC/firetable#887] [Support / Q&A] [answered]
Hello.

Is there please any chance to bulk update fields of existing rows by importing a new csv file?

Thanks. :-)

**Comment by BeeBombshell** [comment]
Hey @michalkasp, bulk updation has been added in one of Rowy's recent releases. Importing data via a `.csv` file is the way to go. I suggest you export the data from your Rowy table as a `.csv` file first. This will incorporate the same column names as the ones in the Rowy table. Thereafter, add data to it and import it back again into your Rowy project. This will let you update the records. 

One thing to keep in mind: for bulk updation, try initializing the data as a `.csv` import as well. This will help match the document id to that in the Firestore collection. If you try to update records that were not added via the import `.csv` method, it will result in duplicates.

**Comment by BeeBombshell** [comment, marked as answer]
Attaching video for reference:

https://user-images.githubusercontent.com/53828745/198545189-5e0c23fc-35a4-4e47-85bb-36f5cf3f9ef7.mp4

**Reply by michalkasp** [reply]
Brilliant! Thank you :-) Now what would be amazing is if I could do the same with subcollections!
